Hec.Data/TimeSeries.cs

```diff
@@ -66,10 +70,16 @@ private static long ToMillisecondsSinceUnixEpoch(DateTime dateTime)
         //https://stackoverflow.com/questions/1376965/when-to-use-a-sortedlisttkey-tvalue-over-a-sorteddictionarytkey-tvalue
         private SortedList<DateTime, TimeSeriesValue> data = new SortedList<DateTime, TimeSeriesValue>();
 
-        public TimeSeries(string id="", string units="")
+        public TimeSeries(string id = "", string units = "")
+        {
+            this.TSID = id;
+            this.Units = units;
+            this.TimeZone = TimeZoneInfo.Utc;
+        } public TimeSeries(string id , string units, TimeZoneInfo  tzInfo)
```

Comment:
nice having optional constructor!    For some use cases defaulting to TimeZoneInfo.Local may be appropriate.